src/test/scala/uk/gov/hmrc/play/json/UnionSpec.scala

```diff
@@ -45,18 +46,18 @@ class UnionSpec extends WordSpec with Matchers {
   "Union.format w/ toJson" should {
 
     "serialise a member of the union type with the correct type field" in {
-      Json.toJson(MemberOne(10)) \ "typeField" shouldBe JsDefined(JsString("ONE"))
-      Json.toJson(MemberTwo("10")) \ "typeField" shouldBe JsDefined(JsString("TWO"))
```

Comment:
This change is needed due to a breaking change in play-json 2.8 where `Writes` was made invariant: https://github.com/playframework/play-json/pull/216

As a result the implicit resolution of `Writes` has changed so it will no longer look for a supertype `Writes` by default.